src/tmpfiles/tmpfiles.c

```diff
@@ -4106,6 +4199,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "image",          required_argument,   NULL, ARG_IMAGE          },
                 { "image-policy",   required_argument,   NULL, ARG_IMAGE_POLICY   },
                 { "replace",        required_argument,   NULL, ARG_REPLACE        },
+                { "dry-run",        no_argument,         NULL, ARG_DRY_RUN        },
```

Comment:
Maybe also add `-n` as a short opt. for this?

Reply:
I like the idea, but let's not do this in this PR. If we add it here, we should add it other places that have `--dry-run`, and it'll turn into a bigger rework.